resources/assets/less/bem/forum-category-header.less

```diff
@@ -111,6 +111,7 @@
     margin-left: 10px;
     display: flex;
     align-items: flex-end;
+    flex-direction: column;
```

Comment:
this probably shouldn't be changed. Also the `titles` element probably need an `align-items: flex-end`

![](https://e.myconan.net/s/2017-01/chrome_2017-01-26_16-21-45.png)